### PR TITLE
Adding user spec test

### PR DESCRIPTION
**Pull request by Istyatur**
Trying to figure out createUser api call

Still working on that api call

/auth/createUser?

Changing url to /api/user

Typo

Typo

Typo

Changing expected error type from 00 to 422 (unprocessable entity)

Debugging

removed log files and cleaned up branch

### CODE REVIEW

File: server/api/user/user.spec.js [hunk 1]
```diff
@@ -19,3 +18,23 @@ describe('GET /api/users/past', function() {
       });
   });
 });
+
+
+describe('POST /api/users/', function() {
+    it('should respond with JSON array', function(done) {
```
Comment (nickkoul) [inline server/api/user/user.spec.js]: I think the exports.create function should respond with a token

**Comment by seveibar** [comment]
@Istyatur can you fix this?

**Comment by Istyatur** [comment]
I lack the knowledge to do so.

**Comment by seveibar** [comment]
Look at the return type for creating a new user

**Comment by seveibar** [comment]
@agundy how did this test even pass?

**Comment by seveibar** [comment]
oh we took out the test condition

File: server/api/user/user.spec.js [hunk 2]
```diff
@@ -19,3 +18,23 @@ describe('GET /api/users/past', function() {
...
+	    "user":{}
+	};
+
+	request(app)
+	    .post('/api/users/')
+	    .send(postData)
+	    .expect(422)
+	    .end(function(err, res) {
+		if(err) return done(err);
+		//res.body.should.be.instanceof(Array);
```
Comment (seveibar) [inline server/api/user/user.spec.js]: let's change this to a todo to get this out of the PR queue

`//TODO check if token present`

File: server/api/user/user.spec.js (hunk 1 above)
Comment (seveibar) [inline server/api/user/user.spec.js]: `it('should respond with a token')`

**Pull request merged**